<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sketchpad</class>
 <widget class="QMainWindow" name="Sketchpad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>901</width>
      <height>750</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuRobot">
    <property name="title">
     <string>Robot</string>
    </property>
    <widget class="QMenu" name="menuWorkspace">
     <property name="title">
      <string>Canvas</string>
     </property>
     <addaction name="actionCreate"/>
     <addaction name="actionLoad"/>
    </widget>
    <widget class="QMenu" name="menuSelect_Robot">
     <property name="title">
      <string>Select Robot</string>
     </property>
     <addaction name="actionCyton"/>
     <addaction name="actionABB"/>
    </widget>
    <addaction name="menuSelect_Robot"/>
    <addaction name="actionConnect"/>
    <addaction name="menuWorkspace"/>
   </widget>
   <widget class="QMenu" name="menuWebcam">
    <property name="title">
     <string>Webcam</string>
    </property>
    <addaction name="actionCalibrate"/>
    <addaction name="actionView"/>
    <addaction name="actionSwitch"/>
    <addaction name="actionJudge"/>
    <addaction name="actionSnap_Webcam_Picture"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Sketch Tools</string>
    </property>
    <addaction name="actionLoad_Photo_Kmeans"/>
    <addaction name="actionLoad_Photo_Canny"/>
    <addaction name="actionSet_sketch_window_size"/>
   </widget>
   <widget class="QMenu" name="menuBrush">
    <property name="title">
     <string>Brush</string>
    </property>
    <addaction name="actionDefine_Shape"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRobot"/>
   <addaction name="menuWebcam"/>
   <addaction name="menuImage"/>
   <addaction name="menuBrush"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionDraw_Line"/>
   <addaction name="actionDraw_Circle"/>
   <addaction name="actionDraw_Square"/>
   <addaction name="separator"/>
   <addaction name="actionActionFill"/>
   <addaction name="actionDraw_Filled_Rectangle"/>
   <addaction name="actionDraw_Filled_Circle"/>
   <addaction name="actionDraw_Filled_Polygon"/>
   <addaction name="separator"/>
   <addaction name="actionLoadPhoto"/>
   <addaction name="actionSnap_Webcam_Picture"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/open.png</normaloff>:/Icon Storage/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/new.png</normaloff>:/Icon Storage/new.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDraw_Square">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/Square.png</normaloff>:/Icon Storage/Square.png</iconset>
   </property>
   <property name="text">
    <string>Draw Square</string>
   </property>
  </action>
  <action name="actionDraw_Circle">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/Circle.png</normaloff>:/Icon Storage/Circle.png</iconset>
   </property>
   <property name="text">
    <string>Draw Circle</string>
   </property>
  </action>
  <action name="actionDraw_Line">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/line.png</normaloff>:/Icon Storage/line.png</iconset>
   </property>
   <property name="text">
    <string>Draw Line</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/save.png</normaloff>:/Icon Storage/save.png</iconset>
   </property>
   <property name="text">
    <string>Save As Project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/save.png</normaloff>:/Icon Storage/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDraw_Filled_Circle">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/FilledCircle.png</normaloff>:/Icon Storage/FilledCircle.png</iconset>
   </property>
   <property name="text">
    <string>Draw Filled Circle</string>
   </property>
  </action>
  <action name="actionDraw_Filled_Rectangle">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/FilledSquare.png</normaloff>:/Icon Storage/FilledSquare.png</iconset>
   </property>
   <property name="text">
    <string>Draw Filled Rectangle</string>
   </property>
  </action>
  <action name="actionDraw_Filled_Polygon">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/FreeForm.png</normaloff>:/Icon Storage/FreeForm.png</iconset>
   </property>
   <property name="text">
    <string>Draw Filled Polygon</string>
   </property>
  </action>
  <action name="actionShutdown">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/OffButton.png</normaloff>:/Icon Storage/OffButton.png</iconset>
   </property>
   <property name="text">
    <string>Shutdown</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/open.png</normaloff>:/Icon Storage/open.png</iconset>
   </property>
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/new.png</normaloff>:/Icon Storage/new.png</iconset>
   </property>
   <property name="text">
    <string>create</string>
   </property>
  </action>
  <action name="actionActionFill">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/fill.png</normaloff>:/Icon Storage/fill.png</iconset>
   </property>
   <property name="text">
    <string>actionFill</string>
   </property>
   <property name="toolTip">
    <string>Fill Region</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/wrench.png</normaloff>:/Icon Storage/wrench.png</iconset>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
   <property name="toolTip">
    <string>launch webcam</string>
   </property>
  </action>
  <action name="actionLoad_Photo_Canny">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/trace.png</normaloff>:/Icon Storage/trace.png</iconset>
   </property>
   <property name="text">
    <string>Define sketch outline (canny)</string>
   </property>
  </action>
  <action name="actionLoad_Photo_Kmeans">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/splatter.png</normaloff>:/Icon Storage/splatter.png</iconset>
   </property>
   <property name="text">
    <string>Split sketch into color regions (kmeans)</string>
   </property>
  </action>
  <action name="actionView">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/eyeball.png</normaloff>:/Icon Storage/eyeball.png</iconset>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="actionSwitch">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/switch.png</normaloff>:/Icon Storage/switch.png</iconset>
   </property>
   <property name="text">
    <string>Switch Webcam</string>
   </property>
  </action>
  <action name="actionJudge">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/gavel.png</normaloff>:/Icon Storage/gavel.png</iconset>
   </property>
   <property name="text">
    <string>Judge</string>
   </property>
  </action>
  <action name="actionSnap_Webcam_Picture">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/webcam.png</normaloff>:/Icon Storage/webcam.png</iconset>
   </property>
   <property name="text">
    <string>Snap Webcam Picture</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/clear.png</normaloff>:/Icon Storage/clear.png</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
   <property name="toolTip">
    <string>clear all</string>
   </property>
  </action>
  <action name="actionCyton">
   <property name="text">
    <string>Cyton</string>
   </property>
  </action>
  <action name="actionABB">
   <property name="text">
    <string>ABB</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/OffButton.png</normaloff>:/Icon Storage/OffButton.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionSet_sketch_window_size">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/resize.png</normaloff>:/Icon Storage/resize.png</iconset>
   </property>
   <property name="text">
    <string>Set sketch window size</string>
   </property>
  </action>
  <action name="actionLoadPhoto">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/camera.png</normaloff>:/Icon Storage/camera.png</iconset>
   </property>
   <property name="text">
    <string>loadPhoto</string>
   </property>
   <property name="toolTip">
    <string>Load Photo</string>
   </property>
  </action>
  <action name="actionDefine_Shape">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/sizeBrush.png</normaloff>:/Icon Storage/sizeBrush.png</iconset>
   </property>
   <property name="text">
    <string>Define Shape</string>
   </property>
   <property name="toolTip">
    <string>Define Brush Shape</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myres.qrc"/>
 </resources>
 <connections/>
</ui>
